org.adempiere.base/src/org/adempiere/model/POWrapper.java

```diff
@@ -145,16 +152,45 @@ else if (methodName.startsWith("is") && (args == null || args.length == 0))
 			if (ii >= 0)
 			{
 				return po.get_Value(ii);
+			}else{
+				// throw IllegalArgumentException in case no default method defined
+				return callDefaultFunction(proxy, method, args, false);
 			}
-			//
-			throw new IllegalArgumentException("Method not supported - "+methodName);
 		}
 		else
 		{
 			return method.invoke(po, args);
 		}
 	}
 	
+	/**
+	 * Call default method of interface class
+	 * 
+	 * @param proxy
+	 * @param method
+	 * @param args
+	 * @param nullIfNotSupported - return null if no default method is defined otherwise throw IllegalArgumentException
+	 * @return
+	 */
+	protected Object callDefaultFunction(Object proxy, Method method, Object[] args, boolean nullIfNotSupported){
+		try {
+			if (method.isDefault())
+				return MethodHandles.lookup()
+		        	.findSpecial(interfaceClass, method.getName(), 
+		        		MethodType.methodType(method.getReturnType(), 
+		        		method.getParameterTypes()), interfaceClass)
+		        .bindTo(proxy)
+		        .invokeWithArguments(args);
+			
+		}catch (Throwable e){
+			throw new IllegalArgumentException("Method not supported - " + method.getName());
+		}
+		if(nullIfNotSupported)

```

Comment:
hi @hieplq , are you sure the code will reach here if the interface doesn't has the method name ?